src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -282,6 +302,18 @@ interface IOptionSettlementEngine {
      */
     function claim(uint256 tokenId) external view returns (Claim memory claimInfo);
 
+    /**
+     * @notice Returns the total amount of options written an exercised for all claims /
+     * option lots created the supplied number of days after the option type is created.
+     * @param optionId The id of the option for the claim buckets.
+     * @param daysAfterOptionTypeCreation The number of days after the creation of the
+     * option type.
+     */
+    function claimBucket(uint256 optionId, uint16 daysAfterOptionTypeCreation)
```

Comment:
Why use relative rather than absolute time here?

Reply:
switched to absolute time (in days)